packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/expiration-modal.tsx

```diff
@@ -40,6 +39,6 @@ const MultipliersExpirationModal = observer(({ is_open, toggleModal }: TMultipli
             </Modal.Footer>
```

Comment:
`style={{ width: '100%', height: '4rem' }} `

Not exactly related to this PR . Is there a need to add inline-style in the Button Component

Reply:
Actually I didn't check, it was there even before TS migration https://github.com/binary-com/deriv-app/pull/10018/files#diff-beae0e06787431ea2146dbab145de1e841f22102ee679feac832371e5b3e572e
<img width="1792" alt="Screenshot 2023-11-23 at 08 46 15" src="https://github.com/binary-com/deriv-app/assets/121025168/ad780126-d19d-4ad7-8e38-f5cdd10ba473">
Will have a look

Reply:
Yep, it can be easily moved to `className` as `<Button />` accept such props (on the screenshot I add `className='dc-btn__wide'` and css for it). But I noticed that even in this file there some others inline-style:
` <div style={{ minHeight: '120px', padding: '1.6rem' }}>` (line 23),
` height='auto' width='calc(100vw - 3.2rem)'` (line 18-19)
<img width="1792" alt="Screenshot 2023-11-23 at 09 32 58" src="https://github.com/binary-com/deriv-app/assets/121025168/370da880-b4eb-46e3-abe7-b5b7e05098db">
A list of files in Trader package from my PR with inline-style without variables inside: _packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/expiration.tsx, packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/expiration-modal.tsx_

The only doubt is should it be done in this PR or in a separate card? As it requires testing
@ahmadtaimoor-deriv @balakrishna-deriv

Reply:
Okay, ny plan is to remove inline styles from two files mentioned above and do a small sanity-check (for multipliers crypto desktop+mobile). This sanity will cover not only styles, but also changes, where I removed 'observer' from expiration-modal

Reply:
Yeah I think a different card should be made for removing inline-styles in other files .

Reply:
I've already done it here for file, which connected with current PR) But for all the rest file from trader package we should do it separately